prefect_docker/scripts/flows/entry_workflows/main_iisg_ingestion.py

```diff
@@ -0,0 +1,29 @@
+import os
+import utils
+from prefect import flow
+from flows.dataset_workflows.iisg_ingestion import iisg_metadata_ingestion
+from flows.workflow_versioning.workflow_versioner import \
+    create_ingestion_workflow_versioning
+
+IISG_METADATA_DIRECTORY = os.getenv('IISG_METADATA_DIRECTORY')
+
+
+@flow
+def iisg_ingestion_pipeline():
+    version = create_ingestion_workflow_versioning(
+        transformer=True,
+        fetcher=True,
+        importer=True,
+        updater=True
+    )
+
+    utils.workflow_executor(
+        iisg_metadata_ingestion,
+        "/local-metadata/iisg-metadata",
```

Comment:
Zou je dit nog in een env willen gooien?

Reply:
Deze kunnen je vervangen met IISG_METADATA_DIRECTORY en dan moet je die even aanpassen in de env